Update Docker steps in Software installs to use containerd (#2742)
astronomer/docs: software/install-azure-standard.md

If you can log in, then your Docker client trusts the registry. If Docker does not trust the Astronomer registry, run the following and restart Docker:

```

> $ mkdir -p /etc/containerd/certs.d
> $ cp privateCA.pem /etc/containerd/certs.d/

Author: ```suggestion
$ mkdir -p /etc/docker/certs.d
$ cp privateCA.pem /etc/docker/certs.d/
```

```
Check the Airflow namespace. If pods are changing at all, then the Houston API trusts the registry.

> If you have Airflow pods in the state "ImagePullBackoff", check the pod description. If you see an x509 error, ensure that you have:

Reviewer: ```suggestion
If you have Airflow pods in the state `ImagePullBackoff`, check the pod description. If you see an x509 error, ensure that you have:
```

> - Configured containerd’s `config_path` to point to `/etc/containerd/certs.d` 

Reviewer: ```suggestion
- Configured containerd’s `config_path` to point to `/etc/containerd/certs.d`.
```

- Added the `privateCaCertsAddToHost` key-value pairs to your Helm chart. 

> If you missed these steps during installation, follow the steps in [Apply a config change](apply-platform-config.md) to add them after installation. If you are using a base image such as CoreOS that does not permit values to be changed, or you otherwise can't update `config.yaml`, please contact Astronomer for additional configuration assistance.

Author: ```suggestion
If you missed these steps during installation, follow the steps in [Apply a config change](apply-platform-config.md) to add them after installation. If you are using a base image such as CoreOS that does not permit values to be changed, or you otherwise can't modify `config.yaml`, contact [Astronomer support](https://support.astronomer.io) for additional configuration assistance.
```
